src/agent/configuration_parser/src/configuration_parser.cpp

```diff
@@ -15,8 +15,13 @@ namespace configuration
         }
         catch (const std::exception& e)
         {
-            std::cout << "Error parsing wazuh.conf file: " << e.what() << std::endl;
-            throw;
+            std::cout << "Using localhost:8080 due to error parsing wazuh.conf file: " << e.what() << std::endl;
+
+            tbl = toml::parse_str(
+                R"([agent]
+                port = "8080"
```

Comment:
Let's name this setting `manager_port`. Apply this change where necessary.

Reply:
done 50ce2a3c5e195be918e8944e3a8376ed1e9a3e40